include/libdnf/advisory/advisory_collection.hpp

```diff
@@ -63,6 +64,7 @@ class AdvisoryCollection {
 private:
     friend Advisory;
     friend AdvisoryQuery;
+    friend AdvisorySet;
```

Comment:
Is this really needed? Seems it's not?

Reply:
Since I have moved the `get_advisory_packages_sorted_by_id` (`get_sorted_advisory_packages`) to `AdvisorySet` it is needed because it uses faster private version of `get_packages(..)` which appends the packages to an input vector (its much faster than returning a new vector for each collection and the coping the packages over).